include/irr/asset/format/EFormat.h

```diff
@@ -1007,12 +1014,27 @@ namespace asset
                 blockByteSize(getTexelOrBlockBytesize(format))
             {}
             
+            //! It converts input texels strides to compute a true extent of an image
+            /*
+                The true extent is a dimension of stride in texels or in blocks, depending
+                of what are you dealing with.
+                @param coord it's a dimension in texels.
+            */
```

Comment:
nope, converts measurements in `texels` to `blocks` while rounding up at the same time

Reply:
adjusted in following commit